src/Compilers/Core/Portable/InternalUtilities/GeneratedCodeUtilities.cs

```diff
@@ -8,94 +8,103 @@
 
 namespace Roslyn.Utilities
 {
-        internal static class GeneratedCodeUtilities
+    internal static class GeneratedCodeUtilities
```

Comment:
That is very handy!

---
In reply to: [119226814](https://github.com/dotnet/roslyn/pull/19875#discussion_r119226814) [](ancestors = 119226814)